### ISSUE TITLE
Widgets do not stretch when screen is stretched

### ISSUE THREAD

**Issue by prjemian** [BCDA-APS/adl2pydm#89]
When a screen is displayed, the widgets do not stretch with the screen window.

As drawn initially:
![as-is](https://user-images.githubusercontent.com/2279984/167962314-26d47cba-9eee-4edd-adcd-4a2408995a4c.png)

Stretched:
![stretched](https://user-images.githubusercontent.com/2279984/167962330-f5c147ae-f387-4004-83ea-87f470e65af1.png)

**Comment by prjemian** [comment]
Stretchability comes from putting widgets into layouts.  Will prototype with the converted `motorxU.ui` screen.  Need to ensure that layouts have zero border size so that visual arrangement is not distorted.

**Comment by prjemian** [comment]
Here is a discussion how to override.  As it says, it won't be easy:

https://forum.qt.io/topic/73488/how-to-automatically-resize-widgets-when-window-is-resized-without-layout/2

**Comment by prjemian** [comment]
Possibly embedding the MEDM screen within a [Qt custom layout](https://doc.qt.io/qt-5/layout.html#how-to-write-a-custom-layout-manager) will satisfy this issue.

**Comment by prjemian** [comment]
On discussion, PyDM has a [`FlowLayout(QLayout)`](https://github.com/slaclab/pydm/blob/5740b1a57c9c7870df26f2d5b898e345a104f423/pydm/widgets/template_repeater.py#L17-L112) that may be a suitable example for starting this one.  Make a new issue and develop in the PyDM repository under `pydm/layouts`.

**Comment by prjemian** [comment]
Once that layout manager is complete, use it when writing screens here.
